website/public/locales/en/chat.json

```
@@ -48,7 +48,7 @@
   "verified_plugin_description": "This plugin has been verified by the Open Assistant team.",
   "view_plugin": "View Plugin",
   "visible_hidden": "Visible & hidden",
-  "warning": "This Assistant is a demonstration version that does not have internet access. It may generate incorrect or misleading information. It is not suitable for important use cases or for giving advice.",
+  "warning": "This Assistant is a demonstration version with limited internet access. It may generate incorrect or misleading information. It is not suitable for important use cases or for giving advice.",
```

Suggest something like this. I would agree with mentioning plugins directly as Abd suggested but the message is already too long on mobile so it's hard to fit in.

```suggestion
  "warning": "This Assistant is a demonstration without full internet access. It may generate incorrect or misleading information. It is not suitable for important use cases or giving advice.",
```